migrations/versions/0411_broadcast_message_history.py

```diff
@@ -38,9 +46,25 @@ def upgrade():
     op.add_column("broadcast_message", sa.Column("submitted_by_id", postgresql.UUID(as_uuid=True), nullable=True))
     op.add_column("broadcast_message", sa.Column("submitted_at", sa.DateTime(), nullable=True))
     op.add_column("broadcast_message", sa.Column("updated_by_id", postgresql.UUID(as_uuid=True), nullable=True))
+    op.create_foreign_key(
+        "broadcast_message_submitted_by_id_fkey",
+        "broadcast_message",
+        "users",
+        ["submitted_by_id"],
+        ["id"],
+    )
+    op.create_foreign_key(
```

Comment:
I might be entirely wrong (I've never used SQLAlchemy before!) but I don't think this is needed if there's `ForeignKeyConstraint` when creating the table. That's only based upon an impression in the other migrations though, I think it's just if adding columns?

Reply:
Nope - I'm talking nonsense, I didn't realise it was for the 'not' `_history` table... 🤦